doc/os-image-poll-and-update.md

```diff
@@ -84,4 +84,17 @@ Or on CRC:
 * oc import-image cirros-is -n openshift-virtualization-os-images --from=kubevirt/cirros-container-disk-demo --scheduled --confirm
 * oc set image-lookup cirros-is -n openshift-virtualization-os-images
 
-More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
\ No newline at end of file
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
+
+## DataImportCron source formats
+
+* PersistentVolumeClaim
+* VolumeSnapshot
+
+DataImportCron has only maintained PVC sources for a long period of time,  
+However, for certain storage types, we know that snapshots provide a better fit.  
+Some examples and more info can be found in [clone-from-volumesnapshot-source](./clone-from-volumesnapshot-source.md).
+
+We keep this provisioner specific knowledge on the [StorageProfile](./storageprofile.md) object for each provisioner under `dataImportCronSourceFormat` (possible values are `snapshot`/`pvc`), which in turns signals the data import cron about which type of source is preferred for a specific provisioner.
```

Comment:
We keep this provisioner-specific information in the [StorageProfile](./storageprofile.md) of the provisioner in `dataImportCronSourceFormat` (possible values are `snapshot`/`pvc`), which tells the DataImportCron which type of source is preferred for the provisioner.

Reply:
:+1: